src/components/card/CardBox.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="cardBox">
+    <h1>CardBox</h1>
+    <h1>
+      {{ cardNumber }}
+    </h1>
+    <button @click="stateController.removeCard(cardNumber)">X</button>
+  </div>
+</template>
+
+<script>
+import { useCardStore } from '../../stores/counter';
+
+export default {
+  setup() {
+    const stateController = useCardStore();
+    return { stateController };
+  },
+  props: {
+    cardNumber: {
+      type:Number
```

Comment:
format the file

Reply:
i formatted file but not saw big difference , i think it problem created prettier ( maybe i used other prettier)